exercises/beer-song/beer_song_test.rb

```diff
@@ -1,69 +1,86 @@
 #!/usr/bin/env ruby
+# encoding: utf-8
 gem 'minitest', '>= 5.0.0'
 require 'minitest/autorun'
 require_relative 'beer_song'
 
+# Common test data version: 9f3d48a
 class BeerSongTest < Minitest::Test
-  def test_the_first_verse
-    expected = "99 bottles of beer on the wall, 99 bottles of beer.\n" \
-      "Take one down and pass it around, 98 bottles of beer on the wall.\n"
+  def test_first_generic_verse
+    # skip
+    expected = <<-TEXT
```

Comment:
Using the `<<-TEXT` means that we can have the ending delimiter indented.

At the moment the ending delimiter is on the 1 column, so this is not needed.  See comment regarding the generator for note.